lib/model.go

```diff
@@ -546,9 +546,50 @@ type ChangesApplyResponse struct {
 	Data []ChangesGetResponseData `json:"data,omitempty"`
 }
 
-
 type UpdateTenantRequest struct {
-	Name 	 string `json:"name"`
-	Data 	 map[string]interface{} `json:"data"`
-	Identifier string `json:"identifier"`
+	Name       string                 `json:"name"`
+	Data       map[string]interface{} `json:"data"`
+	Identifier string                 `json:"identifier"`
+}
+
+type WorkflowGetRequestOptions struct {
+	Page  int `json:"page,omitempty"`
+	Limit int `json:"limit,omitempty"`
+}
+
+type WorkflowData struct {
+	Id                        string        `json:"_id,omitempty"`
+	Name                      string        `json:"name"`
+	Description               string        `json:"description"`
+	Active                    bool          `json:"active,omitempty"`
+	Draft                     bool          `json:"draft,omitempty"`
+	PreferenceSettings        Channel       `json:"preferenceSettings"`
+	Critical                  bool          `json:"critical"`
+	Tags                      []string      `json:"tags"`
+	Steps                     []interface{} `json:"steps"`
```

Comment:
any reason for keeping all these as interface, its ideal to have them as concrete structs gives better DX for the sdk users